source/parse.h

```diff
@@ -5800,9 +5845,6 @@ class parser
     //G     postfix-expression
     //G     prefix-operator prefix-expression
     //GTODO     await-expression
-    //GTODO     'sizeof' '(' type-id ')'
-    //GTODO     'sizeof' '...' ( identifier ')'
-    //GTODO     'alignof' '(' type-id ')'
```

Comment:
Right, these should go anyway as they already work without a special grammar. And _throws-expression_ too, since `throw(...)` works (and I'm using that in `reflect.h2`'s `report_violation` function).